android/widgets/src/main/java/org/nativescript/widgets/Async.java

```diff
@@ -287,7 +288,7 @@ protected RequestResult doInBackground(RequestOptions... params)
 					}
 
 					// Do not attempt to write the content (body) for DELETE method, Java will throw directly
-					if (requestMethod != DeleteMethod)
+					if (!requestMethod.equals(DELETE_METHOD))
```

Comment:
requestMethod defaults to GET_METHOD, don't think it can ever be null either.
https://github.com/Nativescript/tns-core-modules-widgets/blob/master/android/widgets/src/main/java/org/nativescript/widgets/Async.java#L277